adafruit_bitmap_font/ttf.py

```diff
@@ -6,19 +6,20 @@
 # Remove the above when TTF is actually supported.
 
 import struct
-
+from io import FileIO
```

Comment:
Use a `try`/`except` block with `typing` as the first statement per usual for `FileIO` and `Bitmap` imports.